packages/backend/src/server/api/ApiCallService.ts

```diff
@@ -220,11 +220,18 @@ export class ApiCallService implements OnApplicationShutdown {
 	}
 
 	@bindThis
-	private async call(
-		ep: IEndpoint & { exec: any },
+	private async call<Data>(
+		ep: IEndpoint & { exec: (
+			data: Data,
+			user: MiLocalUser,
+			token: MiAccessToken | null | undefined,
+			file: { name: string, path: string } | null,
+			ip: any,
+			headers: any
+		) => Promise<any> },
```

Comment:
```suggestion
	private async call<Data, Ret>(
		ep: IEndpoint & { exec: (
			data: Data,
			user: MiLocalUser,
			token: MiAccessToken | null | undefined,
			file: { name: string, path: string } | null,
			ip: any,
			headers: any
		) => Promise<Ret> },
```

(戻り値も明示的にannotateする必要があるかも？)